apps/condo/domains/organization/constants/common.js

```diff
@@ -0,0 +1,5 @@
+const INN_LENGTH = 10
```

Comment:
Number of digits depends on country in question.
This constant should be a map of countries to number of digits.

Reply:
Created a task SBERDOMA-683